.ci.sh

```diff
@@ -13,6 +13,12 @@ export PIP_PROGRESS_BAR=off # disable pip's progress bar for the duration of CI
 export PY_COLORS=1 # Colored pytest output (https://github.com/pytest-dev/pytest/issues/7443#issuecomment-656642591)
 
 install_sct () {
+  # NB: We want to test the "standalone install" when on master, so copy the install script outside the srcdir
+  if [ "$GITHUB_REF_NAME" == "master" ]; then
```

Comment:
Thanks for this! 

`[` vs. `[[` is a stumbling point for me, so I had just copied and pasted the conditional from another part of the workflow. (I should really learn how to write `bash` without having to fall back to copy-pasting...)

But, now you mention this, a Ctrl+F for `if [` shows that there are quite a few instances of `if [` + `==` for string comparison across other GHA workflows. I guess these issues aren't caught because `shellcheck` can't exactly be run on the bash script parts of GHA workflow files... or can it?

The PyTorch folks were curious about this too (https://github.com/pytorch/pytorch/issues/55314), and they came up with https://github.com/pytorch/pytorch/commit/adc65974b274a009e64cbef40ab59a63e8c0f5dc. I'm kind of curious to try this out!

Reply:
That sounds cool, about the automated shell script extraction to run shellcheck!

Also, I'll note that my nitpick was really a nitpick: `[ "$A" == "$B" ]` works fine in `bash`; it's only a problem in the POSIX `sh`.

Reply:
And I just tested, `shellcheck` only complains about `==` for string comparison when the shebang line identifies the script as `sh`, not when it's `bash`.

Reply:
Ahhhhh, gotcha! Thanks for being thorough about this. :)